pkg/distributor/forwarding/config.go

```diff
@@ -8,11 +8,13 @@ import (
 )
 
 type Config struct {
-	Enabled        bool          `yaml:"enabled" category:"experimental"`
-	RequestTimeout time.Duration `yaml:"request_timeout" category:"experimental"`
+	Enabled         bool          `yaml:"enabled" category:"experimental"`
+	RequestTimeout  time.Duration `yaml:"request_timeout" category:"experimental"`
+	PropagateErrors bool          `yaml:"propagate_errors" category:"experimental"`
 }
 
 func (c *Config) RegisterFlags(f *flag.FlagSet) {
 	f.BoolVar(&c.Enabled, "distributor.forwarding.enabled", false, "Enables the feature to forward certain metrics in remote_write requests, depending on defined rules.")
 	f.DurationVar(&c.RequestTimeout, "distributor.forwarding.request-timeout", 10*time.Second, "Timeout for requests to ingestion endpoints to which we forward metrics.")
+	f.BoolVar(&c.PropagateErrors, "distributor.forwarding.propagate-errors", true, "If disabled then forwarding requests are a fire-and-forget operation, errors get dropped.")
```

Comment:
nit: "fire-and-forget" is a bit jargon-y. Can we be more direct? Maybe something like "If disabled then forwarding requests always succeed, errors are ignored"

Reply:
sure, will update

Reply:
updated it here: https://github.com/grafana/mimir/pull/1978/commits/eb504245fed86c09e063977c656220e8cb1695d9